src/coll_score/ucc_coll_score_map.c

```diff
@@ -87,15 +87,16 @@ static ucc_status_t ucc_coll_score_map_lookup(ucc_score_map_t *map,
     ucc_list_link_t *list;
     ucc_msg_range_t *r;
 
-    if (mt == UCC_MEMORY_TYPE_ASYMMETRIC) {
```

Comment:
Should this still error for non-rooted collectives? Or should I update the PR to copy-out for those too?